insights/parsers/system_time.py

```diff
@@ -174,26 +193,45 @@ class NtpTime(Parser):
 
     """
     def parse_content(self, content):
-        funcname = [
-            'ntp_gettime',
-            'ntp_adjtime'
-        ]
         result = {}
-        return_code_re = re.compile(r'returns code (\d+) ')
-        status_code_re = re.compile(r'\((.*)\)')
+        return_code_re = re.compile(r'(?P<func>ntp_(?:get|adj)time)\(\) returns code (?P<code>\d+) ')
+        status_code_re = re.compile(r'status (?P<status>0x[0-9a-f]{4}) \((?P<flags>.*)\)')
+        time_data_re = re.compile(r'time (?P<timecode>[0-9a-f]{8}\.[0-9a-f]{8}) +(?P<timestamp>\w{3}, \w{3} \d+ .*),')
+        value_re = re.compile(r'(?P<keyword>[A-Za-z][A-Za-z ]+) (?P<value>\d+(?:\.\d+|x[0-9a-f]+)? ?(?:us|ppm|s|\(.*\))?)')
```

Comment:
Dedicated units for output data are well defined. Can we return the numeric value instead? This can ease the latter comparison in plugin. For example:
~~~
assert result['frequency'] == 4.201
assert result['interval'] == 1
assert result['precision'] == 0.001
assert result['tolerance'] == 500
~~~